pkg/model/crd.go

```diff
@@ -1752,7 +1871,10 @@ func (h *Helper) GetCRDs() ([]*CRD, error) {
 			if h.IsIgnoredShape(memberShapeRef.Shape.ShapeName) {
 				continue
 			}
-			memberNames := names.New(memberName)
+			renamedName, _ := crd.InputFieldRename(
```

Comment:
Question: under what conditions does InputFieldRename return false ?

Reply:
it actually returns (string, bool). The string part of the return is the renamed field name, or if there was no renaming done, the original field name.